src/test/java/com/spotify/folsom/client/binary/SetRequestTest.java

```diff
@@ -42,13 +42,22 @@ public void testBufferCas() throws Exception {
     verifySetRequest(258);
   }
 
+  @Test
+  public void testZeroTTL() throws Exception {
+    verifySetRequest(0, 258);
+  }
+
   private void verifySetRequest(long cas) throws Exception {
+    verifySetRequest(1000, cas);
+  }
+
+  private void verifySetRequest(int ttl, long cas) throws Exception {
```

Comment:
I would prefer consistent ordering of arguments (i.e. put cas first like the method above)